lib/_emerge/depgraph.py

```diff
@@ -53,6 +53,15 @@
 from portage.util._eventloop.global_event_loop import global_event_loop
 from portage.versions import _pkg_str, catpkgsplit
 
+try:
+	from portage.xml.metadata import MetaDataXML
+except (SystemExit, KeyboardInterrupt):
+	raise
+except (ImportError, SystemError, RuntimeError, Exception):
```

Comment:
I realize that this is the existing exception handling that we use in EbuildPhase since ee3bf6f9773c0afbd494c657d253241f79d98044, but with the versions of python that we support today (including 2.7) I think it should be sufficient to do it like this:
```python
try:
	from portage.xml.metadata import MetaDataXML
except Exception:
	MetaDataXML = None
```

Reply:
Yeah, I agree, I did check the bug but forgot to follow up on it. I'll fix that shortly.